phac-nml/irida-next
# Workflow Submission: Add custom name to a workflow execution

## What does this PR do and why?
_Describe in detail what your merge request does and why._

Users can add a custom name to a workflow during the submission process.

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

**Light Mode**
![image](https://github.com/phac-nml/irida-next/assets/11295750/113c38be-0ee5-4538-8a60-fb668dce3e54)
 
**Dark Mode**
![image](https://github.com/phac-nml/irida-next/assets/11295750/07938345-020a-4bd2-ab0d-abb5925f5af8)

## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._

1. Open the Workflow Submission dialog
2. There should always be a field at the top for a custom name

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
